sdks/python/apache_beam/metrics/metric.py

```diff
@@ -93,14 +93,57 @@ def update(self, value):
 
 
 class MetricResults(object):
+
+  @staticmethod
+  def _matches_name(filter, metric_key):
```

Comment:
I'll look more closely at the Python implementation after we settle on the Java implementation.